### PR TITLE
Deployment to GitHub Releases

### PR DESCRIPTION
**Pull request by pfigwer**
After commitina a git tag on any branch Travis will build binaries for Linux, Mac and Windows and uploads them to GitHub release as `appliscale-integrations` user.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,4 +7,29 @@ install: make config-install get-deps
...
+  - 'GOOS=windows GOARCH=386 go build -o release/perun-windows-386.exe'
+  - 'tar -C release -czf release/perun-linux-amd64.tar.gz perun-linux-amd64'
+  - 'tar -C release -czf release/perun-darwin-amd64.tar.gz perun-darwin-amd64'
+  - 'tar -C release -czf release/perun-windows-amd64.tar.gz perun-windows-amd64.exe'
+  - 'tar -C release -czf release/perun-windows-386.tar.gz perun-windows-386.exe'
+
+deploy:
+  provider: releases
+  api_key:
+    secure: eLTZravNiDVvQ1dekb7NvWYJBIG2X6CzfHEqUKJ2JCDlKSqYXrvqZF/B3XxmYvmF1tEAmAa08LMDzzzApl9IML1DVSoW8i6uy+uetg+xbvumAf9fq14nMd0JQEEA1qruE7pwjyQs7h9gXYtyAR01CPhj/xNUQmYV1i8NCTHoljBkO+NsMFyi3WMbW7HTRQZQZXbPBagI06L3tSOCfN/w5KVmRsKFQ3lvmnzs+mTrIvOy2CBQC+0Cp3PQ/p7yyhEWRFd5J6n2jYGxneetnBq0FAfbOF4RIwvrWuu9XI/znxhYMOB5lra0qUwuG+prJStB6oaQ/vHStRcxQorV75Jtm4u/EHcFmmaxTQvPksdZQ8VSIbFonz1qbnuurP5sloiAR1RnJQtQWZKj7I7ioknEBh4kqCGvLUIbt0VpHTNoPKN0a8GYiPSE9UO6J+CNS+FR5mahW3xsHx5dHMV+R4mxcbt16dlg0g8m4tah06bd3P/t91kkgliTWmkHDMX4ES4hh+ribMnsLB0k7iqtuoO2P+gFn80CR5ooAX9Z3u8P8MaEovuPSaO7DqsGfX3uCaFInyBpc5EteCNwgN9dGAfh4mscJlijx28qgJ5quNU56fhcfQ8DoC5nXTM7RRRSu0OB1xSDa9OEf5Nh1AlkDwQKxjAYD+ujYFCXxqSWcntUbqE=
```
Comment (morfeush22) [inline .travis.yml]: Is it secure to have key listed here?

Comment (afronski) [reply .travis.yml]: I talked with @pfigwer about it - it is encrypted, and _Travis_ keeps the password on its side, so we are good to have it here.

File: .travis.yml [hunk 2]
```diff
@@ -7,4 +7,29 @@ install: make config-install get-deps
...
+  file:
+    - release/perun-linux-amd64.tar.gz
+    - release/perun-darwin-amd64.tar.gz
+    - release/perun-windows-amd64.tar.gz
+    - release/perun-windows-386.tar.gz
+  skip_cleanup: true
+  on:
+    tags: true
+    all_branches: true
```
Comment (afronski) [inline .travis.yml]: Maybe just on `master`? Do we want to do official releases from other branches?

File: .travis.yml (hunk 1 above)
Comment (pfigwer) [reply .travis.yml]: As @afronski said, it is encrypted.

**Pull request merged**